src/components/control-checkbox-set/control-checkbox-set.tsx

```diff
@@ -0,0 +1,141 @@
+import React, { ReactElement, ReactNode } from 'react';
+import PropTypes from 'prop-types';
+import omit from '../utils/omit';
+import Icon from '../icon';
+import ControlWrapper from '../control-wrapper';
+import { InputProps } from '../typings';
+
+interface Props {
+  id: string;
+  onChange: (value: Array<string>, id: string) => void;
+  options: Array<InputProps & {
+    label: ReactNode;
+  }>;
+  value?: Array<string>;
+  icon?: string;
+  optional?: boolean;
+  autoFocus?: boolean;
+  validationError?: ReactNode;
+  themeCheckboxContainer?: string,
+  themeCheckbox?: string,
+  themeControlWrapper?: string
+}
+
+export default function ControlCheckboxSet({
+  id,
+  onChange,
+  value = [],
+  options,
+  autoFocus = false,
+  icon = 'check',
+  validationError,
+  themeCheckboxContainer = 'txt-s block mb6 flex',
+  themeCheckbox = 'mr6 inline-block checkbox--blue checkbox--s-label',
+  themeControlWrapper
+}: Props): ReactElement {
+
+  const onCheckboxChange = (e) => {
+    const current = e.target.value;
+    const index = value.indexOf(current);
+    const newValue = value;
+
+    if (index === -1) {
+      newValue.push(current);
+    } else {
+      newValue.splice(index, 1);
+    }
+
+    return onChange(newValue, id);
+  };
+
+  const groupProps = {
+    id,
+    'data-testid': `${id}-input`
+  };
+
+  if (validationError) {
+    groupProps['aria-invalid'] = true;
+  }
+
+  const renderOptions = ({ label, ...d }, index: number) => {
+    const isActive = value.indexOf(d.value) >= 0 ? true : false;
```

Comment:
```suggestion
    const isActive = value.includes(value);
```
Ternary expression is redundant here since the return of the function is already boolean